newIDE/app/src/ResourcesEditor/ResourcePropertiesEditor/index.js

```diff
@@ -66,14 +97,24 @@ export default class ResourcePropertiesEditor extends React.Component<
     );
 
     return (
-      <div
-        style={styles.propertiesContainer}
-        key={resources.map(resource => '' + resource.ptr).join(';')}
-      >
-        <PropertiesEditor
-          schema={this.schema.concat(resourceSchema)}
-          instances={resources}
-        />
+      <div key={resources.map(resource => '' + resource.ptr).join(';')}>
+        <Line expand alignItems="center" style={styles.propertiesContainer}>
+          <Column expand>
+            <PropertiesEditor
+              schema={this.schema.concat(resourceSchema)}
+              instances={resources}
+            />
+          </Column>
+          <Column>
```

Comment:
Oh and I thought by looking at the GIF that you added the button as a new field, but it's actually on a column on the right.

I think it's weird to have the button "floating" on the right next to the properties right? It's even taking space of the other fields as it has a whole column for it.
I think we should add it only to the field. I've explained how to do it in: https://github.com/4ian/GDevelop/pull/822#issuecomment-449723035

Let me know if something is not clear.